src/app/app.component.spec.ts

```diff
@@ -1,11 +1,37 @@
 import { TestBed, async } from '@angular/core/testing';
 import { AppComponent } from './app.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { RouterModule } from '@angular/router';
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { SharedModule } from '@app/shared';
+import { CoreModule } from '@app/core';
+import { FileManagerModule } from '@app/file-manager';
+import { SettingsModule } from './settings';
+import { NgxElectronModule } from 'ngx-electron';
+import { RouterTestingModule } from '@angular/router/testing';
+
 describe('AppComponent', () => {
   beforeEach(async(() => {
```

Comment:
Is `async()` needed here, and below in the `it()` clauses? Other examples I've seen don't use this construct.

Reply:
Spec files are created by default by Angular CLI tool so I think follows Angular best practices. I need to do some research before being able to confirm it is a bad practice. Could we put this on testing technical debt ?

Reply:
Yes that would be OK

Reply:
Spec files are created by default by Angular CLI tool so I think follows Angular best practices. I need to do some research before being able to confirm it is a bad practice. Could we put this on testing technical debt ?